src/styles/bds.sass

```diff
@@ -23,4 +23,10 @@
 @import "bds/modifiers.sass"
 
 // Import Bulma extension
-@import "@creativebulma/bulma-tooltip/src/sass/index.sass"
\ No newline at end of file
+@import "@creativebulma/bulma-tooltip/src/sass/index.sass"
+
+body
+    letter-spacing: 0
+
+input, textarea, button, .button
+    border-radius: 4px !important
```

Comment:
What do you think of this @giulio-sl ? I think it could go somewhere else, but I'm not sure where.

Reply:
Good question! I need to investigate this 😅

Reply:
@giulio-sl I ended up moving the global styles to its own file: https://github.com/superlayerhq/biings-ds/pull/13/commits/a9d79549a94c8b03159e97702654dd2d595697e3